src/ix_stresser/pdfbulknode.clj

```diff
@@ -0,0 +1,87 @@
+(ns ix-stresser.pdfbulknode
+  (:require [clj-http.client :as client]
+            [clojure.core.async :as async :refer [<! <!! go go-loop timeout]]
+            [clojure.data.json :as json]))
+
+(def base-files-path "./doc/")
+(def dev-url "http://localhost:3000")
+(def stag-url "http://pdfnode-staging.cloudapp.net")
```

Comment:
where is that? can I add there the pdfNode production/staging urls, keys, and some data to not bet here written?